ContosoHelpdeskChatBot/ContosoHelpdeskChatBot/Dialogs/MainDialog.cs

```diff
@@ -12,17 +12,19 @@ namespace ContosoHelpdeskChatBot.Dialogs
 {
     public class MainDialog : WaterfallDialog
     {
-        private const string InstallAppOption = "Install Application (install)";
-        private const string ResetPasswordOption = "Reset Password (password)";
-        private const string LocalAdminOption = "Request Local Admin (admin)";
-        private const string GreetMessage = "Welcome to **Contoso Helpdesk Chat Bot**.\n\nI am designed to use with mobile email app, make sure your replies do not contain signatures. \n\nFollowing is what I can help you with, just reply with word in parenthesis:";
+        private const string InstallAppOption = "Install Application";
```

Comment:
this is by design to be text so that users can initiate conversation over SMS as well as email